### PR TITLE
chore: event filtering feature test case added with  expected payload for ga

### PR DESCRIPTION
**Pull request by rspriyamdixit**
## PR Description
Added the input and output payload for event filtering.
Please include a summary of the change along with the relevant motivation and context.

## Notion ticket

Ticket link
https://www.notion.so/rudderstacks/JS-SDK-Add-TC-for-feature-Event-filtering-45783bf8b6384764b5472e8e76f14f16?pvs=4
## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: sanity-suite/src/testBookSuites/eventFiltering.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+const eventFilteringSuite = {
+  id: 'eventFilteringFeature',
+  name: 'eventFiltering',
+  description:
+    'Track Method: rudderanalytics.track(event, [properties], [apiOptions], [callback]);',
+  testCases: [
+    {
+      id: 'eventFiltering',
+      description: 'Call with all arguments and all primitives in properties and apiOptions',
+      inputData: [
```
Comment (MoumitaM) [inline sanity-suite/src/testBookSuites/eventFiltering.js:12]: This is not the right payload

**Review by MoumitaM — CHANGES_REQUESTED** [review]

File: sanity-suite/src/testBookSuites/eventFiltering.js (hunk 1 above)
Comment (MoumitaM) [reply sanity-suite/src/testBookSuites/eventFiltering.js:12]: the format should be something like below:
```
inputData:[
"event name here",
{
  key: value // properties
  order_id: "sample order id",
  products:[]
} ,
{
  anotherKey: anotherValue
}
]
```

Comment (MoumitaM) [reply sanity-suite/src/testBookSuites/eventFiltering.js:12]: What you have right now is the SDK-generated payload.

Comment (rspriyamdixit) [reply sanity-suite/src/testBookSuites/eventFiltering.js:12]: @MoumitaM  Pushed the new changes, can you please verify

**Review by MoumitaM — APPROVED** [review]
LGTM

**Review by bardisg — APPROVED** [review]

**Pull request closed without merging**